### PR TITLE
OS X uploader

### PR DESCRIPTION
**Pull request by thomasdenney**
Adds an OS X app that can be used in the same way as the Windows uploader to automatically deploy downloaded HEX files to any connected microbits.

Resolves https://github.com/Microsoft/pxt-microbit/issues/249

### CODE REVIEW

File: clients/macuploader/Microbit Uploader/Uploader.m [hunk 1]
```diff
@@ -0,0 +1,82 @@
+//
+//  Uploader.m
+//  Microbit Uploader
+//
+//  Created by Thomas Denney on 14/09/2016.
```
Comment (pelikhan) [inline clients/macuploader/Microbit Uploader/Uploader.m]: Please remove personal copyrights.

File: clients/macuploader/Microbit Uploader/main.m [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//
+//  main.m
+//  Microbit Uploader
+//
+//  Created by Thomas Denney on 12/09/2016.
```
Comment (pelikhan) [inline clients/macuploader/Microbit Uploader/main.m]: Remove personal copyrights

**Comment by pelikhan** [comment]
Is it possible to add a "pre-built" version of the app in a "release/v..." folder?

**Comment by thomasdenney** [comment]
@pelikhan I can add a pre-built version, although would you prefer it to be unsigned (OS X will show a warning and require people to go into System Preferences to launch it) or signed with my Apple developer account

**Comment by pelikhan** [comment]
Unsigned works.

**Comment by thomasdenney** [comment]
Done.

File: clients/macuploader/Microbit Uploader/Base.lproj/MainMenu.xib [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+                            <menuItem isSeparatorItem="YES" id="kCx-OE-vgT"/>
+                            <menuItem title="Quit Microbit Uploader" keyEquivalent="q" id="4sb-4s-VLi">
+                                <connections>
+                                    <action selector="terminate:" target="-1" id="Te7-pn-YzF"/>
+                                </connections>
+                            </menuItem>
+                        </items>
+                    </menu>
+                </menuItem>
+                <menuItem title="File" id="dMs-cI-mzQ">
```
Comment (pelikhan) [inline clients/macuploader/Microbit Uploader/Base.lproj/MainMenu.xib]: Do we need a file menu?

File: clients/macuploader/Microbit Uploader/Base.lproj/MainMenu.xib [hunk 4]
```diff
@@ -0,0 +1,681 @@
...
+                            </menuItem>
+                            <menuItem title="Print…" keyEquivalent="p" id="aTl-1u-JFS">
+                                <connections>
+                                    <action selector="print:" target="-1" id="qaZ-4w-aoO"/>
+                                </connections>
+                            </menuItem>
+                        </items>
+                    </menu>
+                </menuItem>
+                <menuItem title="Edit" id="5QF-Oa-p0T">
```
Comment (pelikhan) [inline clients/macuploader/Microbit Uploader/Base.lproj/MainMenu.xib]: Do we need a menu at all?

**Comment by thomasdenney** [comment]
MainMenu.xib is pretty much entirely unmodified and never shown. I can delete the file if need be.

**Comment by pelikhan** [comment]
Yes please.

**Comment by pelikhan** [comment]
Can you use the same art as the other uploader?

**Pull request merged**